pkgs/applications/video/anime-downloader/default.nix

```
@@ -0,0 +1,44 @@
+{ lib, python3, aria2, mpv, nodejs, fetchFromGitHub }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "anime-downloader";
+  version = "5.0.14";
+  src = fetchFromGitHub ({
+    owner = "anime-dl";
+    repo = "anime-downloader";
+    rev = version;
+    sha256 = "sha256-Uk2mtsSrb8fCD9JCFzvLBzMEB7ViVDrKPSOKy9ALJ6o=";
+  });
+
+  propagatedBuildInputs = [
+    aria2
+    mpv
+    nodejs
+    python3.pkgs.beautifulsoup4
+    python3.pkgs.cfscrape
+    python3.pkgs.click
+    python3.pkgs.coloredlogs
+    python3.pkgs.fuzzywuzzy
+    python3.pkgs.jsbeautifier
+    python3.pkgs.pySmartDL
+    python3.pkgs.pycryptodome
+    python3.pkgs.pyqt5
+    python3.pkgs.requests
+    python3.pkgs.requests-cache
+    python3.pkgs.selenium
+    python3.pkgs.tabulate
+  ];
```

An alternative, and more readable way to write this:

```suggestion
  propagatedBuildInputs = [
    aria2
    mpv
    nodejs
  ] ++ (with python3.pkgs; [
    beautifulsoup4
    cfscrape
    click
    coloredlogs
    fuzzywuzzy
    jsbeautifier
    pySmartDL
    pycryptodome
    pyqt5
    requests
    requests-cache
    selenium
    tabulate
  ]);
```

---

I was wondering how to do it like that. Thanks for the review!